src/coreclr/src/zap/zapinfo.cpp

```diff
@@ -2268,16 +2268,18 @@ void ZapInfo::getCallInfo(CORINFO_RESOLVED_TOKEN * pResolvedToken,
                 m_zapper->Warning(W("ReadyToRun: Runtime method access checks not supported\n"));
             ThrowHR(E_NOTIMPL);
         }
-
-        if (GetCompileInfo()->IsNativeCallableMethod(pResult->hMethod))
-        {
-            if (m_zapper->m_pOpt->m_verbose)
-                m_zapper->Warning(W("ReadyToRun: References to methods with NativeCallableAttribute not supported\n"));
-            ThrowHR(E_NOTIMPL);
-        }
     }
 #endif
 
+#if defined(TARGET_X86) && defined(TARGET_WINDOWS)
```

Comment:
Could you please add the same block to src\coreclr\src\tools\crossgen2\ILCompiler.ReadyToRun\JitInterface\CorInfoImpl.ReadyToRun.cs ?

Reply:
(throw new NotImplementedException ...)